packages/components/src/lib/theme.tsx

```diff
@@ -0,0 +1,39 @@
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+import type { PaletteRange } from '@mui/joy/styles';
+
+declare module '@mui/joy/styles' {
+  interface ColorPalettePropOverrides {
+    // apply to all Joy UI components that support `color` prop
+    brand: true;
+  }
+
+  interface Palette {
+    // this will make the node `secondary` configurable in `extendTheme`
+    // and add `secondary` to the theme's palette.
+    brand: {
+      blue: string;
+      lightBlue: string;
+    };
+  }
+}
+export const prendaTheme = extendTheme({
+  colorSchemes: {
+    light: {
+      palette: {
+        brand: {
+          blue: '#0A4872',
+          lightBlue: '#D7F3FF',
+        },
+      },
+    },
+  },
+  fontFamily: {
+    body: '"Poppins", sans-serif',
```

Comment:
Poppins is only used for headings. Normal body text is Inter.

Reply:
Right, I was putting something there as a placeholder. Where is all of the body/h1 => font documented?
I looked [here](https://www.figma.com/file/w9lodWeivyS4GHdBNIYaJr/foundations?type=design&node-id=2268-228&mode=design&t=qExCpfiVMmWAFkKU-0)

Reply:
There's no defined mapping because there shouldn't be. See my comment in Figma on yours. Variants and the underlying html tag (component) are rightfully independent.